### DISCUSSION TITLE
[Arcropolis 3.7.2] Dir Addition issue (Many added slots causes CSS instability/break)

### DISCUSSION THREAD

**Discussion by DevorySSBU** [Raytwo/ARCropolis#426] [Issues] [answered]
**System Firmware Version**
14.1.2

**Atmosphere Version**
1.3.1

**Smash Version**
13.0.1

**ARCropolis Version**
3.7.2

**Can the crash be consistently reproduced?**
Yes

**At what moment does the crash happen?**
When idle on the CSS. It will crash after about _5-8 seconds_ each time. 

**Details:**
Seems that this issue is caused by having quite a good amount of mods on added slots. My thoughts are that the rough limit before the game starts to break on the CSS is around _~40 or higher_. Any more, and the CSS might become really unstable as far as it crashing sooner or later. 

You might be fine with 20 added slots, but even then possibly, if it does not crash quickly enough early on, it appears that the more matches you finish, and more chance it might crash as well. 
Either way, it might be an issue somewhere when _on the CSS_.

Sometimes even, it may only crash quickly once you select a fighter.

**Plugins turned on:** 
libarcropolis.nro (for this test, only this plugin exists)

**Crash reports:**
[01663171581_01006a800016e000.log](https://github.com/Raytwo/ARCropolis/files/9568036/01663171581_01006a800016e000.log)
![01663171581_01006a800016e000](https://user-images.githubusercontent.com/96158253/190206709-2f9b1ecf-1ab0-4973-b186-f507c8f53f38.jpg)
[01663181534_01006a800016e000.log](https://github.com/Raytwo/ARCropolis/files/9569106/01663181534_01006a800016e000.log)
![01663181534_01006a800016e000](https://user-images.githubusercontent.com/96158253/190239320-b8846255-aa9a-49c9-beea-545edb383965.jpg)

Reproducible Modpack
[mods.zip](https://github.com/Raytwo/ARCropolis/files/9569117/mods.zip)

**Comment by Raytwo** [comment, marked as answer]
Fixed with ARCropolis 3.8.0!
